www/components/Layout/ProfileButton/AccountModal/AccountModal.js

```diff
@@ -0,0 +1,52 @@
+import { Modal, Avatar } from "@mui/material";
```

Comment:
I actually think that it could be a good idea to create a separate account page at `user/[_id]` (with largely the same format as this modal). That should make it easier for us to allow group members to look at other user's accounts.

Reply:
I tried this initially there was just so much white space I couldn't make it look good. If you think well have enough content to give it its own page we could certainly do it.